cmd/proxy/.env

```diff
@@ -1 +1,3 @@
 ATHENS_MONGO_STORAGE_URL=mongodb://127.0.0.1:27017
+GO_ENV=test_postgres
```

Comment:
I know this is not the main focus of this PR. but can you please include
`TMPDIR=/tmp/` which is needed if the contributor does not have buffalo installed and 
`PATH=${PATH}:./bin` which is needed if somebody installs buffalo using our script

Reply:
@michalpristas would it be ok for us to do this in a follow up?

Reply:
this was addressed in other PRs. please ignore my previous comment